js/simplewebrtc/simplewebrtc.js

```diff
@@ -60,6 +60,16 @@ function SimpleWebRTC(opts) {
 		}
 	}
 
+	// Override screensharing support detection to fit the custom
+	// "getScreenMedia" module.
+	// Note that this is a coarse check; calling "getScreenMedia" may fail even
+	// if "supportScreenSharing" is true.
+	var screenSharingSupported =
+			(window.navigator.mediaDevices && window.navigator.mediaDevices.getDisplayMedia) ||
+			(window.navigator.webkitGetUserMedia) ||
+			(window.navigator.userAgent.match('Firefox'));
+	webrtcSupport.supportScreenSharing = window.location.protocol === 'https:' && screenSharingSupported;
```

Comment:
This seems unnecessary, as we only support https, because otherwise at least chrome does not allow webrtc at all (apart from localhost)

Reply:
This sets `supportScreensharing` to `true` if HTTPS is being used and, in that case, if the browser supports screensharing. What is unnecessary?